packages/zoltan2/src/algorithms/connectivity/Zoltan2_IceProp.hpp

```diff
@@ -0,0 +1,499 @@
+#include "Tpetra_Core.hpp"
```

Comment:
Add Trilinos header boilerplate.

Reply:
I copied the header boilerplate from one of the coloring files.